rapthor/lib/field.py

```diff
@@ -102,8 +102,25 @@ def scan_observations(self, data_fraction=1.0):
             for obs in self.full_observations:
                 mintime = self.parset['calibration_specific']['slow_timestep_sec']
                 tottime = obs.endtime - obs.starttime
+                if data_fraction < min(1.0, mintime/tottime):
+                    obs.log.warning('The specified value of data_fraction ({0:0.3f}) results in a '
+                                    'total time for this observation that is less than the '
+                                    'slow-gain timestep. The data fraction will be increased '
+                                    'to {1:0.3f} ensure the slow-gain timestep requirement is '
```

Comment:
Minor nit-pick: There's a "to" missing. "to {1:0.3f} [to] ensure"

Reply:
Good catch!